### PR TITLE
sing-box 1.1.6 (new formula)

### PR DESCRIPTION
**Pull request by imhansai**
* sing-box 1.1.6 (new formula)
* sing-box: add proxy test
* sing-box: check config

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by imhansai** [comment]
Why should desc be changed? Is "universal proxy platform" OK?
I am not a sing-box collaborator, and I submit a new formula for the first time. Please give me some advice, thank you

**Comment by chenrui333** [comment]
> Why should desc be changed? Is "universal proxy platform" OK? I am not a sing-box collaborator, and I submit a new formula for the first time. Please give me some advice, thank you

> [Correctable] Description shouldn't start with an article.

File: Formula/sing-box.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  desc "The universal proxy platform"
+  homepage "https://sing-box.sagernet.org"
+  url "https://github.com/SagerNet/sing-box/archive/refs/tags/v1.1.6.tar.gz"
+  sha256 "2fdf93fd49c9375cd14b2fe2e2163cbad4b65d0cfa422c592855e7810036ef56"
+  license "GPL-3.0-or-later"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-X \"github.com/sagernet/sing-box/constant.Version=#{version}\" -s -w -buildid="
```
Comment (chenrui333) [inline Formula/sing-box.rb:11]:
```suggestion
    ldflags = "-s -w -X github.com/sagernet/sing-box/constant.Version=#{version} -buildid="
```

**Comment by imhansai** [comment]
[error location](https://github.com/Homebrew/homebrew-core/actions/runs/4411848786/jobs/7730826591#step:7:33) Tips: The homepage URL https://sing-box.sagernet.org/ is not reachable

It is OK for me to test locally,I don't understand how to solve it

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
> [error location](https://github.com/Homebrew/homebrew-core/actions/runs/4411848786/jobs/7730826591#step:7:33) Tips: The homepage URL https://sing-box.sagernet.org/ is not reachable
> 
> It is OK for me to test locally,I don't understand how to solve it

that should not be a blocker.

**Comment by chenrui333** [comment]
@imhansai, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by chenrui333** [comment]
using homebrew_curl would resolve the homepage audit failure issue, it is due to the fact that homepage only allows tlsv1.3 access (while built-in curl on 11 machines does not have that capacity)

**Pull request closed without merging**